### PR TITLE
Add Instructions for how to run app at different ports

### PR DESCRIPTION
**Pull request by alexfinnarn**

### CODE REVIEW

**Comment by alexfinnarn** [comment]
I had to use the binary to pass in the `-p` option instead of `npm run dev -p 3001`, but maybe you can specify a port in that command?

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ $ npm install
 # service worker is disabled in dev
 $ npm run dev
 
+# If port 3000 is already taken, you can specify a port, e.g. port 3001.
+# Assumes you installed Nuxt in the project folder.
+node_modules/.bin/nuxt -p 3001
```
Comment (nyl-auster) [inline README.md]: i this case you can also update your package.json like that
```js
  "scripts": {
    "dev": "PORT=5000 node_modules/.bin/nuxt",
```
The idea is that most of the time, you just want to run "npm run dev" or "npm run start" on all your projects, without asking yourself which is the binary or the exact command to launch the server

Comment (nyl-auster) [reply README.md]: ( but it also forces people working with you to use the same port, not sure about the best practice here )

**Comment by alexfinnarn** [comment]
I think the ideal solution would be for `npm run dev` to try port 3000 out first and then pick other ports based on what is available. I had a previous instance hanging or something else using port 3000 and so I had to specify another port. 

As far as changing the readme text, I think it's all up to the audience of this project. I was able to follow the command in package.json to the binary and then the `--help` for it to see `-p` as the port option. Other users might see the compilation fail and then give up. Adding more info is bad for the more experienced developers (at least I think), but adding all the context you can is helpful for newbies. In this case, the addition doesn't have anything to do with this project, but more so how `nuxt` works. 

This reminds me of math textbooks, kind of. I always wanted more explanation in the answers in the back of the book, but then again, I was never going to be a mathematician, and I'm sure the math people would rather have more problems to work on than fewer problems and more explanations. 

But I can look at some other projects that I feel pick ports dynamically and then just tell you `app is running at localhost:3057`.

**Comment by nyl-auster** [comment]
I agree this is a useful change for the README, was just saying than editing _package.json_ is a most common practice that calling yourself the binary. ( for the reason i gave above ) ( see here the answer of the author of Nuxt for example : https://github.com/nuxt/nuxt.js/issues/149 ) :)

**Comment by nyl-auster** [comment]
i added instruction for port.

**Pull request closed without merging**